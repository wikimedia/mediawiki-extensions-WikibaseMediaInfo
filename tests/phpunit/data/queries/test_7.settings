{
	"title": "phrase search query",
	"term": "\"phrase search\"",
	"entityIdsForTerm": { "\"phrase search\"": [] },
	"userLanguage": "de",
	"fallbackLangs": [],
	"stemmingSettings": {},
	"settings": {
		"boost": {
			"statement": {"P999": 20},
			"descriptions.$language": 7.5,
			"descriptions.$language.plain": 2.5,
			"category": 0.525,
			"category.plain": 0.175,
			"title": 0.45,
			"title.plain": 0.15,
			"redirect.title": 0.375,
			"redirect.title.plain": 0.125,
			"text": 0.3,
			"text.plain": 0.1,
			"auxiliary_text": 0.225,
			"auxiliary_text.plain": 0.075,
			"file_text": 0.15,
			"file_text.plain": 0.05
		}
	}
}
