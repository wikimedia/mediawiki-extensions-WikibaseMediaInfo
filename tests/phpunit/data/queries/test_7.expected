{
    "query_string": {
        "query": "(all.plain:\"phrase search\"~0^1)",
        "fields": [
            "all.plain^1",
            "all^"
        ],
        "phrase_slop": 0,
        "default_operator": "AND",
        "allow_leading_wildcard": false,
        "fuzzy_prefix_length": 2,
        "rewrite": "top_terms_boost_1024"
    }
}